utils/build/webpack.dev.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+const { returnWebpackConfigFor, plugins, CONSTANTS } = require('./webpack.common.js');
```

Comment:
This feels a bit like overkill - could this be achieved by getting the common config and just using object spread to add in new properties?

Reply:
I don't think so (mainly due to how the config is defined in Webpack in arrays and how I want to define as much as possible which is common in that common file) - would be happy to ajust/update if you can come up with something